setup.py

```diff
@@ -53,10 +53,9 @@ def find_version(*parts):
     license='BSD License',
 
     install_requires=[
-        'django>=1.8',
-        'jsonfield>=1.0.3,<3.0.0',
+        'django>=3.1',
```

Comment:
Django 2.2.26 is an LTS, shouldn't we support it?

Reply:
I removed jsonfield package because [JSONField](https://docs.djangoproject.com/en/4.0/releases/3.1/#jsonfield-for-all-supported-database-backends) is supported from Django3.1

Reply:
> I removed jsonfield package because [JSONField](https://docs.djangoproject.com/en/4.0/releases/3.1/#jsonfield-for-all-supported-database-backends) is supported from Django3.1

JSONField only exists in Django3.1 though. Django 2.2 is an LTS and is supported until [April 2022](https://www.djangoproject.com/download/#supported-versions).

So, if we're gonna stop supporting all previous versions, can you add a section on the README saying the previous version is the one they need to use if they want to use something older than Django 3.1?